modules/administration/pages/auth-methods-sso-example.adoc

```diff
@@ -40,6 +40,76 @@ java.sso.onelogin.saml2.idp.single_logout_service.url
+# information about every parameter.
+#
+#
+#
+#
+# If 'strict' is True, then the Java Toolkit will reject unsigned
+# or unencrypted messages if it expects them signed or encrypted
+# Also will reject the messages if not strictly follow the SAML
+#
+# WARNING: In production, this parameter setting parameter MUST be set as "true".
+# Otherwise your environment is not secure and will be exposed to attacks.
+# Enable debug mode (to print errors)
+# Identifier of the SP entity (must be a URI)
+java.sso.onelogin.saml2.sp.entityid = https://suma-long-42-srv.mgr.suse.de/rhn/manager/sso/metadata
+
+# Specifies info about where and how the <AuthnResponse> message MUST be
+# returned to the requester, in this case our SP.
+# URL Location where the <Response> from the IdP will be returned
+java.sso.onelogin.saml2.sp.assertion_consumer_service.url = https://suma-long-42-srv.mgr.suse.de/rhn/manager/sso/acs
+
+# Specifies info about where and how the <Logout Response> message MUST be
+# returned to the requester, in this case our SP.
+java.sso.onelogin.saml2.sp.single_logout_service.url = https://suma-long-42-srv.mgr.suse.de/rhn/manager/sso/sls
+
+# Identifier of the IdP entity (must be a URI)
+java.sso.onelogin.saml2.idp.entityid = http://identity-provider.mgr.suse.de:8080/auth/realms/SUMA
+
+# SSO endpoint info of the IdP. (Authentication Request protocol)
+# URL Target of the IdP where the SP will send the Authentication Request Message
+java.sso.onelogin.saml2.idp.single_sign_on_service.url = http://identity-provider.mgr.suse.de:8080/auth/realms/SUMA/protocol/saml
+
+# SLO endpoint info of the IdP.
+# URL Location of the IdP where the SP will send the SLO Request
+java.sso.onelogin.saml2.idp.single_logout_service.url = http://identity-provider.mgr.suse.de:8080/auth/realms/SUMA/protocol/saml
+
+# Public x509 certificate of the IdP
+java.sso.onelogin.saml2.idp.x509cert = -----BEGIN CERTIFICATE----- MIIClzCCAX8CBgGC+tPbVjANBgkqhkiG9w0BAQsFADAPMQ0wCwYDVQQDDARTVU1BMB4XDTIyMDkwMTIwNTEwNFoXDTMyMDkwMTIwNTI0NFowDzENMAsG
+A1UEAwwEU1VNQTCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBAMNSWJAalB5mShTkMBO5mrsOosyheEL8/A37WvuqDPwwEfm4x0cG7gmMHvONxYXZk+LRyzoQl2sBrNFrbMuwu5dnah5ZSMxQyUu697S280m4vIiegGaFdbgH+g4FGBu
+eSis1ssMzTcES+NUuI7pLkMLNmSQtncESnoL9q2SyeQSwYtr5dz1ydl6IzjwtaWeyQ9EGJNtJtLk3U4+arLPCpHAwqFAnLO9NeYcRDNUKhNBs1v5mHP+L066PZu1/DkE0mSgy/+qXaS0CgZVKqz8qB+bvHVuAq9W60g1CjqZKbwvPu72p/7+d8z
+9DxXPIZ1uxdqn19q/kLEP2TYLtgQobSHECAwEAATANBgkqhkiG9w0BAQsFAAOCAQEAga+raLMJDo/P/yN1Z6SGGocK227WFqovBiE/mLYlp5Ff0+0jS1US1plSppJ94xOr8j0m7HW0Wu5xCz6oOhzXTEtnfIbeRyr1Rms3BWdxyXgQ9bWUeZMWZ
+HfDkTbhgRRmjDEwSSfEXRKQNvw41CpnlB36I0++ejgGnjDvH7BbkCaoW55JF5j6DT/WYR0n7MkEl2Ova9CH0e9X7Gny8iOAg26oziy06uy3P/lx9Z9RmHnvpvN/Q34SGEq9z/HlQVuP12UPj//iT21Jc17OOZFsZQXlGFTG6bXKmO42W8FdUDJU
+ONoXZgjMb3eC7U691YyeowoqTY7mJKxNPprYY/lL0w== -----END CERTIFICATE-----
+
+# Organization
+java.sso.onelogin.saml2.organization.name = Michele Bologna
```

Comment:
For example users, use free-software mascots, such as Tux (Linux Kernel), Wilber (The GIMP), Geeko (SUSE), Foxkeh (Firefox), Konqi (KDE), or Duke (Java).

I would stay away from using personal names in  our examples.